### ISSUE TITLE
Compatibility request - "Rich Soil" from Farmer's Delight

### ISSUE THREAD

**Issue by xFirefalconx** [Pokecube-Development/Pokecube-Issues-and-Wiki#1037]
**Is your feature request related to a problem? Please describe.**
I kind of hate the fact that all sorts of entities can trample fields, that I need to till a piece of land to plant a berry and that I cannot control whether trees grow or berry bushes stay. 

**Describe the solution you'd like**
There's a halfway solution to that. It's called rich soil and it comes from the mod farmer's delight. 
That block can also be used to for example plant sugarcane in large fields. 

What I imagine: When you use that soil type for your berries, you automatically tell the mod that you want to keep small bushes and no tree growth takes place. You can plant berries on the ground for trees, without tilling (vanilla dirt, too). 
This way, you have somewhat control over the bush growth. 

**Describe alternatives you've considered**
Not really that I know any better solutions.

**Additional context**
![Rich Soil](https://user-images.githubusercontent.com/106652912/191336408-d921075f-5f92-4c26-9999-a88aa81f0ff9.PNG)

**Comment by Thutmose** [comment]
I am not sure what you mean by this?

Berries can only be planted on farmland, but they will stay planted on any dirt/grass, so you only need the hoe for initial planting, and after that they stay grown, so trampling is not a concern

**Comment by xFirefalconx** [comment]
Edit: Okay my apologies, it actually works for that type of soil. Can I though somehow control whether the berry grows into a tree or not?

**Comment by Thutmose** [comment]
no the ones that make trees will only make trees, but the berries should have a tooltip for if they grow into a tree or not

**Comment by xFirefalconx** [comment]
Hm, would it then be possible to "pot" plant them? So that they stay relatively small, or so.

**Comment by Thutmose** [comment]
there should be flower pots registered for each berry

**Comment by WenXin20** [comment]
Yes, and placing string on top of the berries that grow into trees will also stop it from doing so.

**Comment by UnLuCkYcHaRrnS** [comment]
That's kind-of a super janky workaround, it would be better if bushes only grew to their final stage as a bush unless bonemealed, at which point they then grow into a tree as normal.

**Comment by WenXin20** [comment]
Its the same as vanilla, string (or any block above) will stop saplings from growing.
